integration-cli/docker_cli_exec_test.go

```diff
@@ -82,7 +82,9 @@ func (s *DockerSuite) TestExecAfterContainerRestart(c *check.C) {
 	testRequires(c, DaemonIsLinux)
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
 	cleanedContainerID := strings.TrimSpace(out)
+	c.Assert(waitRun(cleanedContainerID), check.IsNil)
 	dockerCmd(c, "restart", cleanedContainerID)
+	c.Assert(waitRun(cleanedContainerID), check.IsNil)
```

Comment:
I just wasn't sure if the restart would return immediately or wait for the restart to complete.  No biggie either way

Sent from my iPad

> On Oct 25, 2015, at 2:01 PM, Antonio Murdaca <notifications@github.com> wrote:
> 
> In integration-cli/docker_cli_exec_test.go:
> 
> >  	dockerCmd(c, "restart", cleanedContainerID)
> > +	c.Assert(waitRun(cleanedContainerID), check.IsNil)
> I think it is cause we'll try to exec in a container which could be actually restarting :)
> 
> —
> Reply to this email directly or view it on GitHub.
>